apps/meteor/app/2fa/server/twoFactorRequired.ts

```diff
@@ -6,8 +6,8 @@ import { checkCodeForUser } from './code/index';
 export function twoFactorRequired<TFunction extends (this: Meteor.MethodThisType, ...args: any[]) => any>(
 	fn: TFunction,
 	options?: ITwoFactorOptions,
-): (this: Meteor.MethodThisType, ...args: Parameters<TFunction>) => ReturnType<TFunction> {
-	return function (this: Meteor.MethodThisType, ...args: Parameters<TFunction>): ReturnType<TFunction> {
+): (this: Meteor.MethodThisType, ...args: Parameters<TFunction>) => Promise<ReturnType<TFunction>> {
+	return async function (this: Meteor.MethodThisType, ...args: Parameters<TFunction>): Promise<ReturnType<TFunction>> {
```

Comment:
No changes related making this function async? 👀

Reply:
No need, meteor methods accept promises already.